test/lib/util.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Collection of utils for tests
+
+'use strict'
+
+
+const ORIGINAL_STDOUT_WRITE = process.stdout.write;
+const LOGS_REGEX = /^\[1mfxa-oauth-server/i; // eslint-disable-line no-control-regex
+
+function disableLogs() {
+  // disable fxa-oauth-server logs and others in the future
```

Comment:
Can you leave a note about why this is needed?

Reply:
👍